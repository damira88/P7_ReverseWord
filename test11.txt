The-No.-1-reason-people-fail-in-life-is-because-they-listen-to-their-friends,-family,-and-neighbors.