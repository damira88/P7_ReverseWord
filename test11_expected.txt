Output:ehT-.oN-1-nosaer-elpoep-liaf-ni-efil-si-esuaceb-yeht-netsil-ot-rieht-,sdneirf-,ylimaf-dna-.srobhgien